### ISSUE TITLE
x509 extensions are not included in CSR

### ISSUE THREAD

**Issue by BHoggs** [patrickpr/YAOG#15]
CSR's generated by YAOG don't seem to include the x509 extensions in the CSR. This is fine if you're only using YAOG end-to-end as they get added during signing - but this can be an issue if you want to sign the CSR using an external CA.

![image](https://user-images.githubusercontent.com/40219783/155925471-b2d3a401-2684-457b-bce1-af94b5b74b78.png)

**Comment by chrislaucn** [comment]
Yes I found the same issue, no attributes from x509 extensions are added to the CSR

**Comment by towatai** [comment]
I can Confirm this Issue. Please fix it.
